src/main.js

```diff
@@ -43,15 +43,23 @@ var currentCover;
 
 window.onload = newRandomCover();
 
-
 // Create your event handlers and other functions here 👇
+
 function newRandomCover() {
-  coverImage.src = covers[getRandomIndex(covers)];
-  coverTitle.innerText = titles[getRandomIndex(titles)];
-  taglineOne.innerText = descriptors[getRandomIndex(descriptors)];
-  taglineTwo.innerText = descriptors[getRandomIndex(descriptors)];
+  var coverImg = covers[getRandomIndex(covers)];
+  var coverTitle = titles[getRandomIndex(titles)];
+  var descOne = descriptors[getRandomIndex(descriptors)];
+  var descTwo = descriptors[getRandomIndex(descriptors)];
+  currentCover = new Cover(coverImg, coverTitle, descOne, descTwo);
+  displayCover(currentCover);
 }
 
+function displayCover() {
```

Comment:
Great work! Something we should ask our instructors about is how this displayCover function can be invoked in the above newRandomCover function despite being declared afterwards